src/main/java/com/staffmanagement/controller/UserCrudController.java

```diff
@@ -0,0 +1,67 @@
+package com.staffmanagement.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.staffmanagement.dto.UserDTO;
+import com.staffmanagement.entity.User;
+import com.staffmanagement.service.UserCrudService;
+
+@RestController
+public class UserCrudController {
```

Comment:
```suggestion
public class UserController {
```
just `UserController` should be enough.